novawallet/Modules/Swaps/Setup/View/SwapAssetControl.swift

```diff
@@ -0,0 +1,154 @@
+import UIKit
+import SoraUI
+
+final class SwapAssetControl: BackgroundedContentControl {
```

Comment:
Let's also make this control change opacity (to 0.5) when it is highlighted. Currently there is no feedback on tap.